bin/cml/pr.js

```diff
@@ -39,6 +39,24 @@ exports.builder = (yargs) =>
         conflicts: ['merge', 'rebase'],
         description: 'Try to squash-merge the pull request upon creation.'
       },
+      branch: {
+        type: 'string',
+        description: 'Set a custom branch name for pull request.'
+      },
```

Comment:
What use case do you have in mind for this option? 🤔

Reply:
I was thinking of something slightly more automated and a bit more hacky: set always `--branch` to `cml/{sha256(read(files))}` where `files` is the list of files passed to `cml pr`

Reply:
As an additional change, I agree, but I could still see someone wanting a custom branch. Using this they could still create follow-up logic/actions based on a branch name scheme they already use elsewhere.

What is your concern about adding this?